cms/static/js/views/active_video_upload_list.js

```diff
@@ -198,6 +212,11 @@ define([
                 this.setStatus(data.cid, ActiveVideoUpload.STATUS_FAILED);
             },
 
+            getMaxFileSizeInBytes: function() {
+                var CONVERSION_FACTOR_GBS_TO_BYTES = 1000 * 1000 * 1000;
```

Comment:
I mean why can't we just declare it globally? From the name it seems it is declared as a global variable. `CONVERSION_FACTOR_GBS_TO_BYTES`